src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -70,8 +70,38 @@ public TwilioRestClient(
             string accountSid = null,
             string region = null,
             HttpClient httpClient = null,
-            string edge = null,
-            AuthStrategy authstrategy = null
```

Comment:
you changed the order of the parameters on a public constructor without bumping a major version?

You broke your own library